src/main/java/uk/gov/pay/directdebit/payments/resources/PaymentRequestResource.java

```diff
@@ -50,7 +50,17 @@ public Response getCharge(@PathParam("accountId") String accountExternalId, @Pat
     public Response createNewPaymentRequest(@PathParam("accountId") String accountExternalId, Map<String, String> paymentRequest, @Context UriInfo uriInfo) {
         paymentRequestValidator.validate(paymentRequest);
         LOGGER.info("Creating new payment request - {}", paymentRequest.toString());
-        PaymentRequestResponse response = paymentRequestService.createCharge(paymentRequest, accountExternalId, uriInfo);
+        PaymentRequestResponse response = paymentRequestService.createTransaction(paymentRequest, accountExternalId, uriInfo);
         return created(response.getLink("self")).entity(response).build();
     }
+
+
+    @POST
+    @Path(CANCEL_CHARGE_API_PATH)
```

Comment:
May be we can use `DELETE /v1/api/accounts/{accountId}/payment-requests/{paymentRequestExternalId}` instead of `POST /v1/api/accounts/{accountId}/payment-requests/{paymentRequestExternalId}/cancel` like more restful style? On the other hand if we have `DELETED` status in the future, probably `POST` is fine as it's changing the state, not sure what is better?